retrofit-adapters/rxjava/src/main/java/retrofit2/adapter/rxjava/RxJavaCallAdapterFactory.java

```diff
@@ -37,10 +38,21 @@
    * TODO
    */
   public static RxJavaCallAdapterFactory create() {
-    return new RxJavaCallAdapterFactory();
+    return new RxJavaCallAdapterFactory(null);
   }
 
-  private RxJavaCallAdapterFactory() {
+  /**
+   * TODO
+   */
+  public static RxJavaCallAdapterFactory createWithScheduler(Scheduler scheduler) {
```

Comment:
I'm fighting hard against the need for one in either the converter factories or call adapter factories.